core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -17,26 +17,55 @@
 
 package org.apache.spark.storage
 
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConversions.collectionAsScalaIterable
 import scala.collection.mutable
+import scala.language.reflectiveCalls
```

Comment:
ok, so that is in the StorageStatusListener*Suite*, not StorageStatusListener.  You're getting that b/c you're calling `ticker.advance`, though you haven't explicitly defined a class or interface w/ that method, so scala is using some reflection tricks to do it.  If you change that test as I suggested, that warning will disappear and you can remove the import